src/objclass/class_api.cc

```diff
@@ -677,6 +677,12 @@ int cls_current_subop_num(cls_method_context_t hctx)
   return ctx->current_osd_subop_num;
 }
 
+uint64_t cls_get_features(cls_method_context_t hctx)
```

Comment:
Need to add a dummy implementation of this to test/librados_test_sub/librados/LibradosTestStub.cc